client/rpc/src/lib.rs

```diff
@@ -66,6 +67,52 @@ pub mod frontier_backend_client {
 	use fc_rpc_core::types::BlockNumber;
 	use fp_storage::{EthereumStorageSchema, PALLET_ETHEREUM_SCHEMA};
 
+	/// Implements a default runtime storage override.
+	/// It assumes that the balances and nonces are stored in pallet `system.account`, and
+	/// have `nonce: Index` = `u32` for  and `free: Balance` = `u128`.
+	pub struct DefaultEthereumRuntimeStorageOverride<M>(pub std::marker::PhantomData<M>);
+	impl<M, B, C> fp_rpc::EthereumRuntimeStorageOverride<B, C>
+		for DefaultEthereumRuntimeStorageOverride<M>
+	where
+		M: fp_rpc::EthereumRuntimeAddressMapping + Sync + Send,
+		B: BlockT,
+		C: StorageProvider<B, sc_service::TFullBackend<B>>,
+	{
+		type AddressMapping = M;
```

Comment:
The presence of different address schemes introduces another generic parameter `M` to most of the service build code. We have the option to mark it as concrete in the `EthDeps` config struct to avoid that.